config/cypress.json

```
@@ -6,5 +6,14 @@
   "pluginsFile": "src/platform/testing/e2e/cypress/plugins/index.js",
   "supportFile": "src/platform/testing/e2e/cypress/support/index.js",
   "testFiles": "**/tests/**/*.cypress.spec.js?(x)",
-  "waitForAnimations": false
+  "waitForAnimations": false,
+  "reporter": "cypress-testrail-reporter",
+  "reporterOptions": {
+    "domain": "dsvavsp.testrail.io",
+    "username": "TR_USER",
+    "password": "TR_API_KEY",
```

```suggestion
    "username": "TR_USER",
    "password": "TR_API_KEY",
```
@pjhill , are these values meant to be just temporarily changed for local runs?  If yes, maybe we can abstract the values off to a git-ignored file that Cypress can import at run-time?